tests/pkg-manifest-unlisted-resource/OPS/package.opf

```diff
@@ -12,8 +12,9 @@
     <dc:title>Resource not listed in manifest</dc:title>
     <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
     <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">should</meta>
     <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#confreq-rendition-rs-manifest</meta>
-    <meta property="dcterms:modified">2021-01-08T00:00:00Z</meta>
+    <meta property="dcterms:modified">2021-12-03T00:00:00Z</meta>
```

Comment:
We haven't been updating the dates here; they always match the date above. We could change our strategy, but it seems unnecessarily fussy. I prefer simple and consistent.

Reply:
O.k. I will change it back.